.github/workflows/sphinx-build.yml

```diff
@@ -84,6 +84,8 @@ jobs:
       run: |
         python -m pip install -U pip
         pip install --progress-bar off -U .[document]
+        # TODO(not522): Remove this line after sphinx_rtd_theme supports Sphinx 6
+        pip install -U 'sphinx<6'
```

Comment:
Unfortunately, ReadTheDocs does not refer to settings in `.github`. 
We can confirm it by checking the pre-built document:
https://optuna--4341.org.readthedocs.build/en/4341/